package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "@edx/frontend-learner-portal-base",
+  "description": "Common React components and utility code for building learner portal MFEs.",
+  "version": "1.0.0-semantically-released",
+  "author": "",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "lint": "eslint --ext .js .",
+    "precommit": "npm run lint",
+    "semantic-release": "semantic-release",
+    "test": "jest --coverage"
+  },
+  "dependencies": {
+    "@edx/frontend-component-footer": "^3.0.4",
+    "@edx/frontend-component-site-header": "^2.5.0",
+    "classnames": "^2.2.5",
+    "lodash.camelcase": "^4.3.0",
+    "moment": "^2.24.0",
+    "query-string": "^6.7.0",
+    "reselect": "^4.0.0"
+  },
+  "peerDependencies": {
```

Comment:
It looks like there are some repeat dependencies between `peerDependencies` and `devDependencies`. Is this intentional? For example, `@edx/paragon` is in both categories; should it only be in `peerDependencies`? Similar question for many of these dependencies.

Reply:
Yes, if something is in `peerDependencies` it also needs to be specified in `devDependencies` for testing purposes.